rainforest/tests_test.go

```diff
@@ -520,7 +547,7 @@ func TestUpdateTest(t *testing.T) {
 		}
 	})
 
-	err = client.UpdateTest(&rfTest)
+	err = client.UpdateTest(&rfTest, 0)
```

Comment:
Could we add a comment documenting why this is hardcoded for now?

Reply:
Updated it to use the constant.